go/coordinator/internal/logservice/grpc/server.go

```diff
@@ -0,0 +1,115 @@
+package grpc
+
+import (
+	"context"
+	"errors"
+	"github.com/chroma/chroma-coordinator/internal/grpcutils"
+	"github.com/chroma/chroma-coordinator/internal/logservice"
+	"github.com/chroma/chroma-coordinator/internal/metastore/db/dbcore"
+	"github.com/chroma/chroma-coordinator/internal/proto/logservicepb"
+	"github.com/pingcap/log"
+	"go.uber.org/zap"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/health"
+)
+
+type Config struct {
+	// GrpcConfig config
+	GrpcConfig *grpcutils.GrpcConfig
+
+	// System catalog provider
+	DBProvider string
+
+	// Postgres config
+	DBHost     string
```

Comment:
nit: would be nice to move this into nested struct if possible

Reply:
sure